mw-webapp/src/logic/aboutProjectPage/AboutProjectPage.tsx

```diff
@@ -18,25 +80,83 @@ export const AboutProjectPage = () => {
 
   return (
     <div className={styles.pageWrapper}>
-      <Title
-        level={HeadingLevel.h2}
-        text={LangService.aboutProject.mainTitle[language]}
-      />
+      <div className={styles.aboutBlock}>
+        <Title
+          level={HeadingLevel.h2}
+          text={LangService.aboutProject.mainTitle[language]}
+          className={styles.title}
+        />
+        <Title
+          level={HeadingLevel.h3}
+          text={LangService.aboutProject.mainSubTitle[language]}
+          className={styles.subtitle}
+        />
+        <div className={styles.aboutDescription}>
+          <div className={styles.aboutTextBlock}>
+            <div>
+              {LangService.aboutProject.descriptionSubTitle[language]}
+            </div>
+            <Title
+              level={HeadingLevel.h3}
+              text={LangService.aboutProject.descriptionTitle[language]}
+            />
+            <div>
+              {renderMarkdown(
+                LangService.aboutProject.descriptionList[language],
+              )}
+            </div>
+          </div>
+          <Logotype
+            className={styles.logoAbout}
+            isColorSecondary
+          />
+        </div>
+      </div>
+
+      <div className={styles.ourTeamBlock}>
+        <Title
+          level={HeadingLevel.h2}
+          text={LangService.aboutProject.AboutTeamTitle[language]}
+        />
 
-      <div className={styles.projectDescription}>
-        {renderMarkdown(LangService.aboutProject.projectDescription[language])}
+        <div className={styles.ourTeamMembers}>
+          {teamMembers.map((member) => {
+            return (
+              <div
```

Comment:
let's create a component to represent members

Reply:
created